race_test.go

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Dgraph Labs, Inc. and Contributors
+ * Modifications copyright (C) 2020 Andy Kimball and Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package arenaskl
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TestNodeArenaEnd tests allocating a node at the boundary of an arena. In Go
+// 1.14 when the race detector is running, Go will also perform some pointer
+// alignment checks. It will detect alignment issues where a node's memory would
+// straddle the arena boundary, with unused regions of the node struct dipping
+// into unallocated memory. This test is only run when the race build tag is
+// provided.
```

Comment:
Looks like the `// +build race` build constraint got dropped when this test was copied.

Reply:
Done.